.eslintrc

```diff
@@ -5,4 +5,12 @@
     "max-len": 0,
     "no-console": 0,
   },
+  "overrides": [{
+    "files": ["test/**/*.js"],
+    "rules": {
+      "no-undef": "off",
```

Comment:
I think instead of disabling these we should set the env to jest. https://eslint.org/docs/user-guide/configuring#specifying-environments

Reply:
I can get rid of the no-undef override with this env setting, but not the other two rules unfortunately.